src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        return new Ball(ColorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
Directly returning the new object is good, but since ColorSupplier.getRandomColor() is likely static based on the context, you should also consider creating an instance of ColorSupplier in the Lottery class instead of using a static method.